src/controllers/TaskController.ts

```diff
@@ -0,0 +1,129 @@
+import { Request, Response } from "express";
+import { getRepository } from "typeorm";
+import { validate } from "class-validator";
+ 
+import { Task } from "../entities/Task";
+import { Project } from "../entities/Project";
+
+class TaskController {
+  static getAll = async (req: Request, res: Response) => {
+    const projectId: number = req.params.projectId;
+    const taskEntityRepo = getRepository(Task);
+    const tasks = await taskEntityRepo.find();
+
+    res.send(tasks);
+  };
+
+  static getOneById = async (req: Request, res: Response) => {
+    const id: number = req.params.id;
+
+    const taskEntityRepo = getRepository(Task); 
+    const task = await taskEntityRepo.findOne(id, {
+      select: ["id", "task", "completed"]
+    });
+
+    if(!task) {
+      res.status(404).send({ error: `Task with id ${id} not found` });
+      return;
+    }
+
+    res.send(task);
+  };
+
+  static createTask = async (req: Request, res: Response) => {
+    const projectId = req.params.projectId; 
```

Comment:
You are not checking if a `project` with provided `projectId` exists or not in our records (db).